src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/support/SupportController.java

```diff
@@ -96,6 +101,23 @@ public void resendStaffNotifications(
         }
     }
 
+    @PutMapping("/claim/event/{event}/resend-multiple-notifications")
+    @ApiOperation("Resend notifications for multiple claims associated with the event provided")
+    public void resendMultipleNotifications(
```

Comment:
Just have endpoint called resendClaimsToRPA without any event or switch around it. We'll refactor it later if we require more functionality which is event driven.